### ISSUE TITLE
[resize plugin] make minimal size when resizing configurable

### ISSUE THREAD

**Issue by damianatorrpm** [WayfireWM/wayfire#488]
If wayfire SSD is used (mostly xwayland), the minimal size of a window should be the available buttons + 3-4 letters space as width and height of the window deco buttons.

Currently it is possible if the application does not have itself minimum size to resize the window until it becomes almost un-targetable, basically a square

**Comment by soreau** [comment]
This could be an option added to resize plugin for minimum width/height.

**Comment by ammen99** [comment]
Yeah I think it currently limits to 10x10 or something like that, which is too small.
